### PR TITLE
Add Property 'sheet' on type 'SVGStyleElement'

### PR DESCRIPTION
**Pull request by Daliak**
This is a first attempt to fix this issue: https://github.com/microsoft/TypeScript/issues/36896

### CODE REVIEW

**Review by DanielRosenwasser — APPROVED** [review]
Looks good, but I don't know 100% how this repo works.

File: inputfiles/browser.webidl.preprocessed.json [hunk 1]
```diff
@@ -74396,6 +74396,13 @@
                             "type": "DOMString",
                             "content-attribute-reflects": 1
                         },
+                        "sheet": {
```
Comment (sandersn) [inline inputfiles/browser.webidl.preprocessed.json:74399]: This is a generated file too, so this change should go in addedTypes.json.

(The format there is slightly different there, I think, unfortunately.)

**Review by sandersn — CHANGES_REQUESTED** [review]
This produces the right output but the code change needs to go in addedTypes.json

**Comment by saschanaz** [comment]
#922 fixed this.

**Pull request closed without merging**